.travis.yml

```diff
@@ -2,23 +2,50 @@ language: php
 
 sudo: false
 
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+env:
+    global:
+        - TEST_COMMAND="composer test"
 
 matrix:
-  include:
-    - php: 5.5
-      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+    fast_finish: true
+    include:
+          # Minimum supported PHP and Symfony version
+        - php: 5.5
+          env: DEPENDENCIES="minimum"
 
-before_script:
-  - git config --global user.name travis-ci
-  - git config --global user.email travis@example.com
+          # Test the latest stable release
+        - php: 7.1
+        - php: hhvm
+
+          # Test LTS versions
+        - php: 5.6
+          env: DEPENDENCIES="symfony/lts:v2"
+        - php: 7.0
+          env: DEPENDENCIES="symfony/lts:v3"
+
+          # Latest beta release
+        - php: 7.1
+          env: DEPENDENCIES="beta"
```

Comment:
it uses `3.4-beta`, not `4.0@dev`

perhaps, this would be useful: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.8/.travis.yml#L70

Reply:
That is strange.. It picks 4.0 beta of other symfony packages.

Reply:
@nicolas-grekas, This is a false positive since composer is not using 4.0-beta. 

Maybe we need an symfony/lts-strict after all...

Reply:
I'm :+1: with whole heart for it
(I would go even further and have only strict, but I could understand other point of view as well)

Reply:
@Nyholm , if you would put `DEPENDENCIES=dev`, wouldn't composer pick up `4.0@dev` ?

Reply:
No. Right now I do “beta” and it does not pick it up somehow. Using “dev” would be the same. 

I’ll verify this later. I’m about to board a plane now

Reply:
then just for reference, we we able to enforce installing v4 using dev and in-place composer.json manipulation:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.8/.travis.yml#L70
(but it was still in time when beta was not yet available)